### ISSUE TITLE
Using cordova plugins

### ISSUE THREAD

**Issue by xypaul** [poetic/ember-cli-cordova#142]
I've installed the cordova camera plugin with:
```ember cordova plugin add org.apache.cordova.camera```

However when I try to call the function in JS to open the camera I get undefined for ```navigator.camera```.

Is there anything special I have to do to make this work?

**Comment by dukex** [comment]
try  [cordova-plugin-camera](https://github.com/apache/cordova-plugin-camera) example:

```
document.addEventListener("deviceready", onDeviceReady, false);
function onDeviceReady() {
    console.log(navigator.camera);
}
```

I tested here and worked fine! your code wait deviceready?

**Comment by xypaul** [comment]
Yeah my deviceready isn't firing as shown in the screenshot below. I am using the iOS simulator and debugging through Safari remote debugger.

![screenshot 2015-05-11 16 37 34](https://cloud.githubusercontent.com/assets/2549828/7559574/2a368b98-f7fc-11e4-97ba-45cfb17c58ef.png)

**Any idea why this might be happening?**

Also @dukex if you could share the code where you go it working, that would really help me out thanks :)

**Comment by xypaul** [comment]
Problem solved. Plugins weren't working with live reload turned on.

**Comment by greis** [comment]
This was fixed on master by #137. If you change the dependency ember-cli-cordova to point to master then live reload will work again with plugins.

**Comment by xypaul** [comment]
@greis how do I do that?

My package.json looks like this:
![screenshot 2015-05-12 13 23 04](https://cloud.githubusercontent.com/assets/2549828/7579635/140414e0-f8aa-11e4-9d21-9397e72a9223.png)

**Comment by dukex** [comment]
Change ember-cli-cordova to 
``` 
"ember-cli-cordova": "poetic/ember-cli-cordova#master",
```

or 

```    
"ember-cli-cordova": "https://github.com/poetic/ember-cli-cordova.git#master",
```

**Comment by xypaul** [comment]
@dukex Too be honest I don't understand why that would produce different results. Thanks though :)

**Comment by dukex** [comment]
@xypaul my examples is about @greis comment to change ember-cli-cordova to get current code from master branch not the 0.0.16 version

**Comment by DanielOchoa** [comment]
@greis @xypaul I've published 0.0.17 in npm. You should now be able to set your package.json to use it.

**Comment by xypaul** [comment]
@DanielOchoa thanks
